subprojects/platform-jvm/build.gradle.kts

```diff
@@ -50,6 +50,7 @@ dependencies {
 }
 
 strictCompile {
+    ignoreRawTypes() // raw types used in public API
```

Comment:
Are there few enough of these that we could ignore them in place and not turn this on for the whole subproject?

Reply:
A much better idea. There were only a couple places to add an annotation